model_compression_toolkit/core/common/quantization/quantization_params_generation/uniform_selection.py

```diff
@@ -56,7 +56,8 @@ def uniform_selection_tensor(tensor_data: np.ndarray,
     if quant_error_method == qc.QuantizationErrorMethod.NOCLIPPING:
         mm = tensor_min, tensor_max
     else:
-        error_function = get_threshold_selection_tensor_error_function(QuantizationMethod.UNIFORM, quant_error_method, p, norm=False)
+        axis = -1 if per_channel else None
```

Comment:
I don't think this is the case for Pytorch. I think you need to use the fw_info

Reply:
numpy